blazor/common/showcase-samples.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: Syncfusion Blazor Components showcase samples | Syncfusion
+description: Learn here all about list of showcase samples in Blazor Server and WebAssembly (WASM) apps and much more.
+platform: Blazor
+component: Common
+documentation: ug
+---
+
+# Syncfusion Blazor Components showcase samples
+
+Syncfusion has a collection of sample applications that demonstrate the use of Syncfusion Blazor UI components.
+
+## Blazor Server samples
+
+* [Expense Tracker](https://blazor.syncfusion.com/showcase/expensetracker/)
+* [Diagram Builder](https://blazor.syncfusion.com/showcase/diagrambuilder/)
+* [Floor Planner](https://blazor.syncfusion.com/showcase/floor-planner/)
+* [Logic Circuit Designer](https://blazor.syncfusion.com/showcase/logic-circuit-designer/)
+* [Appointment Planner](https://blazor.syncfusion.com/showcase/appointmentplanner/)
+* [Mind Map](https://blazor.syncfusion.com/showcase/mind-map/)
+* [Bpmn Editor](https://blazor.syncfusion.com/showcase/bpmn-editor/)
+* [Organizational Chart](https://blazor.syncfusion.com/showcase/organization-chart/)
+* [Document Explorer](https://blazor.syncfusion.com/showcase/documentexplorer/)
+* [Hotel Booking](https://blazor.syncfusion.com/showcase/hotel-booking)
+* [Health Tracker](https://blazor.syncfusion.com/showcase/wasm/health-tracker/)
```

Comment:
move to wasm section @SubathraKaliamoorthy

Reply:
moved to wasm section